apps/email/js/cards/message_list.html

```diff
@@ -33,8 +33,9 @@
          class="msg-listedit-header collapsed" role="region">
   <header>
     <a href="#" data-event="click:setEditModeDone"
-       class="msg-listedit-cancel-btn">
-      <span class="icon icon-close"></span>
+       class="msg-listedit-cancel-btn" role="button">
+      <span class="icon icon-close" data-l10n-id="message-listEdit-close">
```

Comment:
Put a label on the anchor (button) itself instead. Also change the name to 'close-button' as it is pretty general and we might have to reuse it elsewhere.